docs/Hero.md

```diff
@@ -99,17 +99,19 @@ Any other props supplied will be provided to the root element (`div`).
 
 You can override the classes used by Brightlayer UI by passing a `classes` prop. It supports the following keys:
 
-| Name | Description                        |
-| ---- | ---------------------------------- |
-| root | Styles applied to the root element |
+| Name   | Description                          |
+| ------ | ------------------------------------ |
+| root   | Styles applied to the root element   |
+| icon   | Styles applied to the icon element   |
+| label  | Styles applied to the label element  |
+| values | Styles applied to the values element |
```

Comment:
why is there a mix of plural and singular

Reply:
I see your point, but this was in there prior to the changes made in this PR. This is out of scope for this issue and changing it now would be a breaking change. Please write a story to fix this in a separate milestone if you believe it to be an issue.

Reply:
The hero component can also take in multiple children, which would make the plural make sense since its styling all "values" whereas there will never be more than one label or icon.